labs/sysdev-kernel-fetch-sources-stm32mp2/sysdev-kernel-fetch-sources-stm32mp2.tex

```diff
@@ -0,0 +1,62 @@
+\subchapter{Fetching Linux kernel sources}{Objective: learn how to fetch
+the Linux kernel sources from git, from both the master and stable
+branches.}
+
+After this lab, you will be able to:
+\begin{itemize}
+\item Get the kernel sources from Stmicroelectronics git.
+\end{itemize}
+
+\section{Setup}
+
+Create the \code{$HOME/__SESSION_NAME__-labs/kernel} directory and go into it.
+
+Since the Linux kernel git repository is huge, our goal here is to start
+downloading it right now, before starting the lectures about the Linux
+kernel.
+
+\section{Cloning the Linux tree}
+
+To begin working with the Linux kernel sources, we need to clone its
+reference git tree, the one managed by Stmicroelectronics.
+
+However, this requires downloading more than 2.8 GB of data. If you
+are running this command from home, or if you have very fast access to
+the Internet at work (and if you are not 256 participants in the
+training room), you can do it directly by connecting to
+\url{https://git.kernel.org}:
+
+\begin{verbatim}
+git clone https://github.com/STMicroelectronics/linux
```

Comment:
I think we should tell people to clone the Linus Torvalds tree, and then add the ST tree as a remote.